dockstore-common/generated/src/main/resources/pom.xml

```diff
@@ -19,7 +19,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>io.dockstore</groupId>
   <artifactId>dockstore-common</artifactId>
-  <version>1.11.0-rc.1-SNAPSHOT</version>
+  <version>1.11.1-SNAPSHOT</version>
```

Comment:
Why did all of these change? I guess it doesn't matter as the maven release process will address it.

Reply:
1.11.0 is the current release anyway, so 1.11.1-SNAPSHOT seems appropriate